src/utils/hooks/useGetResponsesAndElementState.tsx

```diff
@@ -7,13 +7,17 @@ import {
   TemplateElement,
   Application,
 } from '../generated/graphql'
+import { useUserState } from '../../contexts/UserState'
```

Comment:
Interesting! I didn't know you would have access to the context in here. That's cool!

Reply:
Yes, we can have Application state here too if we want it, so that can also be used in evaluation expressions. :)